addons/binding/org.openhab.binding.homie/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/org.eclipse.paho.client.mqttv3-1.1.0.jar,\
```

Comment:
@martinvw

Hello, do you have an example of how to use this service? I'm having troubles creating a binding that relies on this service and i can't seem to find it used in any other OH2 binding, nor can I find docs on the matter.
Also, see https://www.eclipse.org/forums/index.php/t/1088038/